glide.yaml

```diff
@@ -8,6 +8,7 @@ import:
 - package: github.com/gorilla/context
 - package: github.com/gorilla/mux
 - package: github.com/heketi/rest
+  version: aa6a652074133e6b0836bb095e847021fcb94b6a
```

Comment:
This is a remnant of the original patch in this PR.
The required change (to glide.lock) got merged in PR #1133.
This hunk is not needed.
@Madhu-1 please remove this patch.